HotSpot/View Controllers/SignUpViewController.m

```diff
@@ -0,0 +1,120 @@
+//
+//  SignUpViewController.m
+//  HotSpot
+//
+//  Created by aaronm17 on 7/16/19.
+//  Copyright © 2019 aodemuyi. All rights reserved.
+//
+
+#import "SignUpViewController.h"
+#import "Parse/Parse.h"
+
+@interface SignUpViewController ()
+
+@property (weak, nonatomic) IBOutlet UITextField *username;
+@property (weak, nonatomic) IBOutlet UITextField *password;
+@property (weak, nonatomic) IBOutlet UITextField *fullName;
+@property (weak, nonatomic) IBOutlet UITextField *phoneNumber;
+
+@end
+
+@implementation SignUpViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+}
+
+//registers a user, throws up an alert with an "empty" error message if one of the fields is empty.
+//if there's a different error, it throws up an alert with the specific error description
+- (void)registerUser {
+    PFUser *newUser = [PFUser user];
+    
+    newUser.username = self.username.text;
+    newUser.password = self.password.text;
+    
+    //will be initialized once changes in app delegate are pulled
+    //newUser.fullName = self.fullName.text;
+    //newUser.phoneNumber = self.phoneNumber.text;
+    
+    UIAlertController *alert = [UIAlertController alertControllerWithTitle:@"Sign Up Error"
+                                                                   message:@"" preferredStyle:UIAlertControllerStyleAlert];
+
+    // create an OK action
+    UIAlertAction *okAction = [UIAlertAction actionWithTitle:@"OK"
+                                                       style:UIAlertActionStyleDefault
+                                                     handler:^(UIAlertAction * _Nonnull action) {
+                                                         // handle response here.
+                                                     }];
+    // add the OK action to the alert controller
+    [alert addAction:okAction];
+    
+    /*once phone number functionality is working on login, will check
+    if there are letters present in the phone number and throw up an alert
```

Comment:
for the phone number text field, you can launch a number pad instead of a full keyboard. this would ensure that the user can only type in numbers